charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -891,3 +891,175 @@ opAMPBridge:
 
     # Rules for the bridge cluster role
     rules: []
+
+############################
+# Prometheus Configuration #
+# (optional)               #
+############################
+# This configuration sections allows for a direct replacement of the kube-prometheus-stack
+# chart where the collector scrapes the same metrics as the default prometheus installation.
+
+## Component scraping kube state metrics
+##
+kubeStateMetrics:
+  enabled: false
```

Comment:
Are we able to move `enabled` under `kube-state-metrics` or do they have a json schema that wont allow it?

Reply:
that may work, this is mostly for kube-prom-stack compatibility which matches this.

Reply:
Like you could copy from your values.yaml into this one?

Reply:
[yes](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L2024-L2027), i think there's some value in doing that for compatibility, but I think this is overall cleaner...

Reply:
Ok ya I'm fine with keeping it 100% compatible.